### DISCUSSION TITLE
How to use kor-toggle, I can't get the change of the value

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [kor-ui/kor#117] [Q&A] [answered]
kor-ui is one of the most beautiful WebComponent ui libs,  So I select it . 

I used the stencil to develop the component. The code is as follow.

```html
<kor-toggle onchange={this.korToggleChange}></kor-toggle>
```

I can't monitor the change event. The func is not called. 

I go to read the source code of kor-ui, get nothing.

How can I get the value of kor-toggle, how to monitor the change of the value ?

**Comment by Linya-IronMan** [comment]
onChange is also unuseful

**Comment by eduferfer** [comment, marked as answer]
> kor-ui is one of the most beautiful WebComponent ui libs, So I select it .
> 
> I used the stencil to develop the component. The code is as follow.
> 
> ```
> <kor-toggle onchange={this.korToggleChange}></kor-toggle>
> ```
> 
> I can't monitor the change event. The func is not called.
> 
> I go to read the source code of kor-ui, get nothing.
> 
> How can I get the value of kor-toggle, how to monitor the change of the value ?

Hi, the components fire events according to the property name changed. For example, toggle will fire `active-changed` when toggled on or off. 

You can find more info on that here: https://kor-ui.com/introduction/data-binding
